src/styles/WordTestList.scss

```
@@ -0,0 +1,70 @@
+@import './variables/colors.scss';
+@import './variables/textStyles.scss';
+
+.babo {
+    display: flex;
+    justify-content: center;
+    border-bottom: 1px solid;
+    padding-bottom: 16px;
+    margin-bottom: 12px;
+    width: calc(100vw - 40px);
+    margin-left: auto;
+    margin-right: auto;
+    color: $koreasy-color-gray-03;
+}
+
+.test-button {
+    @include body01-bold;
+    background-color: $koreasy-color-primary;
+    color: $koreasy-color-white;
+    border: none;
+    margin-right: 12px;
+    margin-left: 20px;
+    width: calc((100vw - 52px) / 2);
+    height: 40px;
+    border-radius: 12px;
+    font-size: medium;
+}
+
+.learn-button {
+    @include body01-bold;
+    border: none;
+    background-color: $koreasy-color-gray-06;
+    color: $koreasy-color-primary;
+    width: calc((100vw - 52px) / 2);
+    margin-right: 20px;
+    height: 40px;
+    border-radius: 12px;
+    font-size: medium;
+}
+
+.word-test-list-view-name {
+    @include title01-heavy;
+    color: $koreasy-color-primary;
+    margin-left: 20px;
+    // margin-bottom: 12px;
+}
+
+.word-test-main-container {
+    display: flex;
+    flex-direction: column;
+}
+
+.word-test-list-title {
```

```suggestion
.word-test-list-title {
    margin: 0 20px 12px 20px;
```
`big-title`에 있는 `margin-left: 20px`가 `small-title`에 있는 `margin-left: 20px`와 중복됩니다.

이럴 경우에는 중복된 소스코드를 지양하기 위해 `<div>`로 묶어준 다음에 한 번에 간격을 주는 것을 권장합니다.
`margin`을 사용할 경우 上-右-下-左 (상-우-하-좌) 순으로 간격을 주게됩니다.

이미 위쪽으로는 `babo`라는 클래스를 통해 12px를 주었으므로 0으로 두어야 합니다.
다만, 좌측-우측에는 20px를 줘야하므로 우와 좌 값으로 20px를 설정해주었습니다.
하단에는 12px 공백을 줘야하므로 하 값으로도 12px를 주었습니다.